src/main/java/redis/clients/jedis/ScanParams.java

```diff
@@ -61,7 +58,7 @@ int count() {
         if (params.containsKey(COUNT)) {
             return params.get(COUNT).getInt();
         } else {
-            return SCAN_COUNT_DEFAULT.getInt();
+            return 0;
```

Comment:
Let's have consistent between match() and count(). I think it's better to change return type to Integer (to let users know it can be null), and also return null when it's not presented by user.